serverless/openvino/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openvino/ubuntu20_runtime:2022.3.0
+FROM openvino/ubuntu22_runtime:2023.3.0
```

Comment:
Why not use the latest version here?

Reply:
I don't want to change the current functions to use the new API because https://github.com/openvinotoolkit/openvino/issues/18348,
so I changed the version to the latest LTS, which supports API 1.0.